pages/common/n.md

```diff
@@ -0,0 +1,24 @@
+# n
+
+> Node version management.
+> Node.js version management: no subshells, no profile setup, no convoluted API, just simple.
```

Comment:
I see that you have copied this verbatim from the repo home page. We try to be as objective as possible. Unless, its really adding value to the first description, there's no need to add a second line.